js/sites.js

```diff
@@ -203,11 +203,42 @@
       let enhanced = this.link;
       let url = this.url;
 
+      var getRegularThumbnail = async(function*() {
```

Comment:
we're gonna need to do a re-write of sites.js in order to use async functions in the correct scope...since right now we're in the scope of the prototype and it's behaving in a way we don't want to. we'll have to fix this up. perhaps in a separate bug?